src/app/components/core/user-management/user-management.component.ts

```diff
@@ -90,17 +100,44 @@ export class UserManagementComponent implements OnInit, OnDestroy {
   }
 
   getAccessMethod(accessMethod) {
+    let method;
     switch (accessMethod) {
       case 'cert':
-        return 'Certificate';
+        method = 'Certificate';
         break;
       case 'pwd':
-        return 'Password';
+        method = 'Password';
         break;
       default:
-        return 'Any';
+        method = 'Any';
         break;
     }
+    return method;
+  }
+
+  calculateBlockUserTime(time: string): string {
```

Comment:
You don't need this; As in System alert manager, Similarly We can use https://momentjs.com/docs/#/displaying/tonow/